buildkite/bazelci.py

```diff
@@ -3458,6 +3458,12 @@ def main(argv=None):
 
             platform = get_platform_for_task(args.task, task_config)
 
+            # The value of `BUILDKITE_MESSAGE` defaults to the commit message, which can be too large
+            # on Windows, therefore we truncate the value to 1000 characters.
+            # See https://github.com/bazelbuild/continuous-integration/issues/1218
+            if "BUILDKITE_MESSAGE" in os.environ:
```

Comment:
Are there any other environment variables that could be a problem? Looking at the failing build step (https://buildkite.com/bazel-trusted/bazel-release/builds/305#4397c7bb-e13f-4268-85b6-7d37fee966bb), both BUILDKITE_COMMAND and BUILDKITE_SCRIPT_PATH look fairly long, and I don't think our tooling needs them.

Reply:
`BUILDKITE_COMMAND` has a much lower chance to go over the 32768 character length limit and it might be helpful in future (eg. for debugging reason). So I would keep it for now.